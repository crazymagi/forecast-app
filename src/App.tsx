import Container from '@mui/material/Container';
import Grid2 from '@mui/material/Unstable_Grid2';
import { Report } from './components/report';
import './App.css';

function App() {
  return (
    <div className="App">
      <Container maxWidth="lg">
        <Grid2 margin={20}>
          <Report latitude={44.44} longitude={44.44} days={7}/>
        </Grid2>
      </Container>
    </div>
  );
}

export default App;
