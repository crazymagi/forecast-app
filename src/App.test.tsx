import React from 'react';
import { render, screen } from '@testing-library/react';
import App from './App';

jest.mock("./components/report", () => ({
  Report: () => {
    return <div data-testid="report"/>;
  },
}));

test('renders matches snapshot', () => {
  const {asFragment} = render(<App />)
  expect(asFragment()).toMatchSnapshot();
});

test('renders the report', () => {
  render(<App />);
  const reportElement = screen.getByTestId('report');
  expect(reportElement).toBeInTheDocument();
});
