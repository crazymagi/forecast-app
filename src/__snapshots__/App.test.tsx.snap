// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        class="MuiGrid2-root MuiGrid2-direction-xs-row css-8x7r3w-MuiGrid2-root"
      >
        <div
          data-testid="report"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
